apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3219,13 +3219,20 @@ static void tc_libc_stdio_tempnam(void)
 	const char filename[] = "tempnam_tc_test";
 	char *ret1 = NULL;
 	char *ret2 = NULL;
-	int ret_check = 0;
 	FILE *fp;
 
 	/* TC 1 with valid prefix */
-
+#if defined CONFIG_FS_SMARTFS && !defined CONFIG_FS_AUTOMOUNT_TMPFS
```

Comment:
Should you create a separate file for testcases related to tmp fs just like proc fs test case file ?

Reply:
These testcases are for libc/stdio api which requires /tmp mount point ,these are not testing the complete tmpfs . So I think no need  to create a separate file for these testcases.